Formula/n/nauty.rb

```diff
@@ -61,3 +65,17 @@ def install
     system "./test"
   end
 end
+__END__
+diff --git a/makefile.in b/makefile.in
+index 422ff69..572448f 100644
+--- a/makefile.in
++++ b/makefile.in
+@@ -17,7 +17,7 @@ exec_prefix=@exec_prefix@
+ bindir=@bindir@
+ libdir=@libdir@
+ includedir=@includedir@
+-pkgconfigexecdir=${prefix}/libdata/pkgconfig
++pkgconfigexecdir=${libdir}/pkgconfig
+ 
+ INSTALL=@INSTALL@
+ INSTALL_DATA=@INSTALL_DATA@
```

Comment:
does this get sent to the upstream?

Reply:
Yes, I have suggested this patch to upstream.
(Upstream does not have a public VCS repo, so I described it in prose)

Reply:
I'll email upstream with a link to this PR